lib/msal-browser/src/telemetry/PerformanceManager.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Logger } from "@azure/msal-common";
+import { PerformanceMeasurement } from "./PerformanceMeasurement";
+
+export class PerformanceManager {
+    private logger: Logger;
+    
+    constructor(logger: Logger) {
+        this.logger = logger;
+    }
+
+    startMeasurement(measureName: string): Function {
+        this.logger.trace(`Performance measurement started for ${measureName}`);
+        const performanceMeasure = new PerformanceMeasurement(measureName);
+        performanceMeasure.startMeasurement();
+
+        return () => {
```

Comment:
```suggestion
        return (): number => {
```

Reply:
Actually this should return more than just the number, probably an object with number, name, correlation id, etc.